corebehrt/modules/setup/directory_causal.py

```
@@ -96,7 +96,10 @@ def setup_train_mlp(self) -> None:
         self.write_config(
             "trained_mlp", source="calibrated_predictions", name=CALIBRATE_CFG
         )
-        self.write_config("trained_mlp", source="cohort", name=COHORT_CFG)
+        try:
+            self.write_config("trained_mlp", source="cohort", name=COHORT_CFG)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)